docs/standard/serialization/system-text-json-configure-options.md

```diff
@@ -46,6 +46,9 @@ There is a [JsonSerializerOptions constructor](xref:System.Text.Json.JsonSeriali
 
 :::code language="csharp" source="snippets/system-text-json-how-to-5-0/csharp/CopyOptions.cs" highlight="28":::
 :::code language="vb" source="snippets/system-text-json-how-to-5-0/vb/CopyOptions.vb" :::
+
+The metadata cache of the existing `JsonSerializerOptions` instance isn't copied to the new instance. So using this constructor is not the same as reusing an existing instance of `JsonSerializerOptions`.
```

Comment:
What is the "metadata cache"? I don't think I've ever heard of it before.

Reply:
It's explained in the preceding section:

>The serializer undergoes a warm-up phase during the first serialization of each type in the object graph when a new options instance is passed to it. This warm-up includes creating a cache of metadata that is needed for serialization. The metadata includes delegates to property getters, setters, constructor arguments, specified attributes, and so forth. This metadata cache is stored in the options instance. The same warm-up process and cache applies to deserialization.